modules/ingester/instance.go

```diff
@@ -15,7 +16,7 @@ import (
 	"github.com/gogo/status"
 	"github.com/google/uuid"
 	"github.com/opentracing/opentracing-go"
-	"github.com/pkg/errors"
+	pkgErrors "github.com/pkg/errors"
```

Comment:
any particular reason we're adding this? .New, .Wrap, and .Wrapf all appear in the standard `errors` package

Reply:
That was already there. When i imported the standard `errors` package, I had to rename that particular `errors` package. But I could also just not import the standard package and just use what's already there, which is that `pkg/errors`.

Reply:
oh, i see. it would be my inclination to just move everything to `errors`